proto/worker.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "github.com/dmitsh/gravitest/proto";
+
+service Worker {
+  rpc StartProcess (StartProcessRequest) returns (JobId);
+  rpc GetProcessStatus (JobId) returns (Status);
+  rpc StreamOutput (JobId) returns (stream LogLine) {}
+  rpc StopProcess (JobId) returns (Empty);
+}
+
+message JobId {
+  uint64 id = 1;
+}
+
+message Status {
+  string status = 1;
```

Comment:
I would recommend to replace `string` type with `enum` where job statuses are listed explicitly.